wiki/Contests/SVS/1/en.md

```diff
@@ -0,0 +1,91 @@
+- [Discord server](https://discord.gg/8WbX3XdmTs)
+- [Results sheet](https://docs.google.com/spreadsheets/d/1ketuj4BgsevTpKVIM16d7drEdLYYYntPNdX2uIHfpk4/edit?usp=sharing)
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | User |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_NL] [Cookie_Tree](https://osu.ppy.sh/users/502722) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_MY] [\[Zeth\]](https://osu.ppy.sh/users/9912966) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_JP] [Grape_Tea](https://osu.ppy.sh/users/9540073) |
+
+## Ruleset
+
+### Contest rules
+
+- **The contest will consist of one round where contestants will map any song of their choosing.** The chosen song must not breach the osu! [song content rules](/wiki/Rules/Song_Content_Rules); if you wouldn’t upload it to your profile, then don’t use it here. Anything that violates the osu! rules will be disqualified.
+- **Each contestant is limited to a single difficulty of any level.**
+- **Your map must be an osu!taiko map.**
+- **Every entry should utilize only timing points (BPM changes) and inherited points (slider velocities).** NThe maps don’t need to be rankable; in fact, It's to give up on any chance of making a rankable map. If it helps, think of this like Aspire, but incredibly simplified. All entries that include storyboarding, hitsounding, and skinning will be disqualified. You may use sliders and spinners. Fringe entries will be dealt with on a case-by-case basis.
+- **Maps must not be uploaded before results are announced.** This includes maps that were created with no intention of being entered in this contest. Anybody who shares their map in any way before the announcement post is up will be disqualified from the contest.
+- **Submissions must be submitted via .osz format.** Sending in `.osu` files only is not allowed. If you don’t know how to export a .osz file, please follow [this guide](/wiki/Client/File_formats). All `.osu` files submitted will be disqualified if a .osz is not sent afterwards.
+- **Submissions will not be accepted after the submission deadline.** No exceptions. Only the most recent entry will be considered.
+- **In order to participate, you must join the [Discord server](https://discord.gg/8WbX3XdmTs) for the contest and register as a participant.** This is purely for administrative purposes in order to keep track of who is who and make the contest run smoothly.
+- **Songs must be under 5 minutes in length.** Judging requires maps to be played multiple times, this rule is just to be respectful of their time.
+
+### Judging criteria
+
+Judging will be based on a holistic and generic scoring method. Since creativity is incredibly subjective and the placements of this contest are in no way indicative of objective quality, this contest has three loosely-defined categories that contestants should aim to fulfill:
```

Comment:
```suggestion
Judging will be based on a holistic and generic scoring method. Since creativity is incredibly subjective and the placements of this contest are in no way indicative of objective quality, this contest has three loosely defined categories that contestants should aim to fulfill:
```